examples/ads.amp.html

```diff
@@ -689,6 +690,23 @@ <h2 id="ibillboard">iBillboard 300x250 banner</h2>
       src="https://go.eu.bbelements.com/please/code?j-21414.1.5.6.0.0._blank">
   </amp-ad>
 
+  <h2 id="imedia">Imedia</h2>
+  <amp-ad width="300" height="250"
+      type="imedia"
+      data-id="p1"
+      data-positions='[{"id":"p1", "zoneId":"seznam.novinky.ikona2"}, {"id":"p2", "zoneId":"seznam.novinky.ikona"}]'>
+      <div placeholder></div>
```

Comment:
no need for placeholder and fallback here with our default placeholder/fallback supported now.